inc/class-command.php

```diff
@@ -70,6 +70,13 @@ class Command {
 	 *
 	 * [--format=<format>]
 	 * : Render output in a particular format.
+	 *
+	 * [--order=<order>]
+	 * : Ascending or Descending order. ASC|DESC.
+	 *
+	 * [--orderby=<orderby>]
+	 * : Order by fields.
```

Comment:
What's the default order?

Reply:
The default `order` is `ASC` and there is no default `orderby`